### PR TITLE
Updating bundle installer to use Node.js (ChakraCore)

### PR DESCRIPTION
**Pull request by munyirik**

### CODE REVIEW

**Comment by munyirik** [comment]
@kunalspathak @alanch-ms please CR.

File: Setup/BundleInstaller/NTVSBundleInstaller.wxs [hunk 1]
```diff
@@ -13,7 +13,7 @@
     </BootstrapperApplicationRef>	
 	
     <Chain>
-      <MsiPackage Id="NodejsChakraInstaller" SourceFile="$(env.NodejsToolsBundleRelDir)\node-chakra.msi" Vital="yes" DisplayInternalUI="yes" Visible="yes" ForcePerMachine="yes"/>
+      <MsiPackage Id="NodejsChakraInstaller" SourceFile="$(env.NodejsToolsBundleRelDir)\node-chakracore.msi" Vital="yes" DisplayInternalUI="yes" Visible="yes" ForcePerMachine="yes"/>
```
Comment (kunalspathak) [inline Setup/BundleInstaller/NTVSBundleInstaller.wxs]: Stupid question - Isn't the file arch specific ? like node-chakracore-x86.msi, node-chakracore-x64.msi?

Comment (munyirik) [inline Setup/BundleInstaller/NTVSBundleInstaller.wxs]: Hmm, good point. We've been installing x86 by default so far. I think it's possible to install x86 or x64 based on the system. I'll look into that.

**Comment by alanch-ms** [comment]
:+1:

File: Setup/BundleInstaller/NTVSBundleInstaller.wxs (hunk 1 above)
Comment (munyirik) [inline Setup/BundleInstaller/NTVSBundleInstaller.wxs]: @kunalspathak I've fixed this.

**Comment by kunalspathak** [comment]
:+1:

**Pull request merged**